R/use_import_from.R

```diff
@@ -49,6 +49,7 @@ use_import_from <- function(package, fun, load = is_interactive()) {
 }
 
 check_fun_exists <- function(package, fun) {
+  check_installed(package)
```

Comment:
maybe this should go prior to using `check_fun_exists()`, since it will unnecessarily call `check_installed()` multiple times if `length(fun) > 1`